File: solutions/72-rafidhaque/src/Rectangle.java
import javafx.scene.shape.Line;
import javafx.scene.shape.Shape;

import java.util.ArrayList;

public class Rectangle {
    private double x1, y1, x2, y2;
    public Line line1, line2, line3, line4;

    public Rectangle(double startX, double startY, double endX, double endY) {
        this.x1 = startX;
        this.y1 = startY;
        this.x2 = endX;
        this.y2 = endY;
        line1 = new Line(x1, y1, x1, y2);
        line2 = new Line(x1, y1, x2, y1);
        line3 = new Line(x2, y1, x2, y2);
        line4 = new Line(x1, y2, x2, y2);
    }

    public ArrayList<Shape> draw() {
        ArrayList<Shape> arrayList = new ArrayList<>();
        arrayList.add(line1);
        arrayList.add(line2);
        arrayList.add(line3);
        arrayList.add(line4);
        return arrayList;
    }


}

File: solutions/13-akib/Age_Of_Villagers/AgeOfVillagers/AgeOfVillagers/Rectangle.cs
﻿namespace AgeOfVillagers
{
    internal class Rectangle
    {
        private int x1, y1, x2, y2, x3, y3, x4, y4;
        public Rectangle(int x1, int y1, int x2, int y2, int x3, int y3, int x4, int y4)
        {
            this.x1 = x1;
            this.y1 = y1;
            this.x2 = x2;
            this.y2 = y2;
            this.x3 = x3;
            this.y3 = y3;
            this.x4 = x4;
            this.y4 = y4;
        }
        public string Draw_Shape()
        {
            Lines line1 = new Lines(x1, y1, x2, y2);
            Lines line2 = new Lines(x2, y2, x3, y3);
            Lines line3 = new Lines(x3, y3, x4, y4);
            Lines line4 = new Lines(x4, y4, x1, y1);

            return line1.Draw_Shape() + line2.Draw_Shape() + line3.Draw_Shape() + line4.Draw_Shape();
        }
    }
}